api/src/main/java/io/minio/ErrorCode.java

```diff
@@ -32,6 +32,98 @@
   RESOURCE_CONFLICT("ResourceConflict", "Request resource conflicts"),
   RETRY_HEAD_BUCKET("RetryHeadBucket", "Retry HEAD bucket request"),
 
+  // MinIO server specific error codes
+  POST_POLICY_INVALID_KEY_NAME(
+      "PostPolicyInvalidKeyName", "Invalid according to Policy: Policy Condition failed"),
+  UNSUPPORTED_NOTIFICATION(
+      "UnsupportedNotification",
+      "MinIO server does not support Topic or Cloud Function based notifications."),
+  X_GCS_INTERNAL_ERROR("XGCSInternalError", "Error occurred at Google cloud storage."),
+  X_MINIO_ADMIN_BUCKET_QUOTA_DISABLED(
+      "XMinioAdminBucketQuotaDisabled",
+      "Quota specified but disk usage crawl is disabled on MinIO server"),
+  X_MINIO_ADMIN_BUCKET_QUOTA_EXCEEDED("XMinioAdminBucketQuotaExceeded", "Bucket quota exceeded"),
+  X_MINIO_ADMIN_BUCKET_REPLICATION_DISABLED(
+      "XMinioAdminBucketReplicationDisabled",
+      "Replication specified but disk usage crawl is disabled on MinIO server"),
+  X_MINIO_ADMIN_CONFIG_BAD_JSON(
+      "XMinioAdminConfigBadJSON", "JSON configuration provided is of incorrect format"),
+  X_MINIO_ADMIN_CONFIG_DUPLICATE_KEYS(
+      "XMinioAdminConfigDuplicateKeys",
+      "JSON configuration provided has objects with duplicate keys"),
+  X_MINIO_ADMIN_CONFIG_NO_QUORUM(
+      "XMinioAdminConfigNoQuorum", "Configuration update failed because server quorum was not met"),
+  X_MINIO_ADMIN_CONFIG_TOO_LARGE(
```

Comment:
@balamurugana, since this can change especially XMinioAdmin, is not part of S3 API why do we need to add this? 

Why not make non-translatable errors become ErrorResponse instead? instead of this type of hardcoded value - especially the error description for our Admin API errors can change anytime.

Reply:
Better idea is to remove `ErrorCode.java` itself. It would ease out for any S3 services.

Reply:
Yeah agreed @balamurugana